src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -765,11 +765,15 @@ public function queue_import( $args = [] ) {
 
 			$error = $response;
 
+			tribe( 'logger' )->log_debug( "Error during the queue of the record.", 'EA Queue Import' );
```

Comment:
:no_entry_sign: String "Error during the queue of the record." does not require double quotes; use single quotes instead

<sub>hash: 7cc4729875d0da074f2b671a42c92c84</sub>